gateway-js/src/loadSupergraphSdlFromStorage.ts

```diff
@@ -38,6 +38,57 @@ const { name, version } = require('../package.json');
 
 const fetchErrorMsg = "An error occurred while fetching your schema from Apollo: ";
 
+var fetchCounter: number = 0;
+
+export class SupergraphSdlResult {
+  id: string;
+  supergraphSdl: string;
+  constructor(id: string, supergraphSdl: string) {
+    this.id = id;
+    this.supergraphSdl = supergraphSdl;
+  }
+}
```

Comment:
I don't think you want a class here, just an interface. Also, this interface already exists, so you should be able to just import it 👍 

https://github.com/apollographql/federation/blob/59f93a864e7d24541e6e7cf565ea76a536917a9c/gateway-js/src/config.ts#L78-L81

Reply:
perfect! I now use `SupergraphSdlUpdate` instead of my new class, which is now deleted.